reloj-checador/db/seeds/admins/admins.rb

```diff
@@ -1,18 +1,22 @@
+require "faker"
+
 admins = [
-  { name: "Admin 1", last_name: "Admin Lastname 1", email: "admin1@email.com", password: "pass12345" },
-  { name: "Admin 2", last_name: "Admin Lastname 2", email: "admin2@email.com", password: "pass12345" },
-  { name: "Admin 3", last_name: "Admin Lastname 3", email: "admin3@email.com", password: "pass12345" },
+  { email: "admin1@email.com", password: "pass12345" },
+  { email: "admin2@email.com", password: "pass12345" },
+  { email: "admin3@email.com", password: "pass12345" },
 ]
 
+p "Creating admins"
+
 admins.each { |admin|
-  puts "Adding #{admin[:name]} #{admin[:last_name]} #{admin[:email]} #{admin[:password]}"
+  # puts "Adding #{admin[:email]} #{admin[:password]}"
```

Comment:
If you don't need it, you can delete it.

Reply:
Sugestion taken. Code changed; line deleted